db_dtypes/__init__.py

```diff
@@ -122,15 +150,18 @@ def astype(self, dtype, copy=True):
         else:
             return super().astype(dtype, copy=copy)
 
-    if pandas_release < (1,):
+    def __arrow_array__(self, type=None):
+        array = pyarrow.array(self._ndarray, type=pyarrow.timestamp("ns"))
 
-        def to_numpy(self, dtype="object"):
-            return self.astype(dtype)
+        # ChunkedArray has no "view" method, so combine into an Array.
+        array = array.combine_chunks() if hasattr(array, "combine_chunks") else array
```

Comment:
Same here.
```suggestion
        if isinstance(array, ChunkedArray):
            array = array.combine_chunks()
```

Reply:
I kept this one as a ternary because I can't figure out how to force the numpy -> chunkedarray conversion without onerous mocking.